tests/test_ec2/test_network_acls.py

```diff
@@ -385,6 +385,21 @@ def test_describe_network_acls():
     )
     resp4["NetworkAcls"].should.have.length_of(0)
 
+    # Ensure filtering by egress flag
+    resp4 = conn.describe_network_acls(
+        Filters=[{"Name": "entry.egress", "Values": ["false"]}]
+    )
+    resp4["NetworkAcls"].should.have.length_of(3)
```

Comment:
These tests are executed in parallel in ServerMode, so Moto is also returning NetworkACL's created in tests. That's why the tests are failing, it's returning much more than 3 ACL's.

The best way to deal with this IMO, is to only assert whether the response (does not) contains a specific ACL - the specific length is irrelevant.

Reply:
thanks for the clarification! updated the asserts